core/store/migrations/migration1585918589/migrate.go

```diff
@@ -24,7 +24,7 @@ func Migrate(tx *gorm.DB) error {
 
 	// FIXME: This should ideally be unique but there exists non-unique data out in the wild so need to handle that...
 	err = tx.Exec(`
-	CREATE INDEX idx_txs_hash ON tx_attempts(hash);
+	CREATE INDEX idx_txs_hash ON txes(hash);
```

Comment:
This was a mistake. Since no release has occurred since the last merge, should be fine to modify the migration.